model_loader.py

```diff
@@ -314,18 +314,50 @@ def _UpdateSensors(self, software_version, sensors):
             sensor.supports_recording = recording
             save = True
 
+        new_normal_min = sensor_info.get('normal_min')
+        if new_normal_min is not None and sensor.normal_min != new_normal_min:
+          sensor.normal_min = new_normal_min
+          save = True
+        new_normal_max = sensor_info.get('normal_max')
```

Comment:
Do you want to add a blank lie before each new value please, to match the existing code.